rune/libenclave/internal/runtime/pal/skeleton/Makefile

```diff
@@ -3,15 +3,27 @@ OBJCOPY := $(CROSS_COMPILE)objcopy
 endif
 
 OUTPUT ?= ./
+SGX_SDK ?= /opt/intel/sgxsdk
+RA_TLS := ../../../../../../ra-tls
+BINDIR := $(RA_TLS)/build/bin
+LIBDIR := $(RA_TLS)/build/lib
+INCDIR := $(RA_TLS)/build/include
+SGX_RA_TLS := $(RA_TLS)/sgx-ra-tls
+RA_TLS_SERVER := $(RA_TLS)/ra-tls-server
+
 HOST_CFLAGS := -Wall -Werror -g -fPIC -z noexecstack \
-	       -Wno-unused-const-variable -std=gnu11 -I../kvmtool/include
+	       -Wno-unused-const-variable -std=gnu11 -I../kvmtool/include \
+	       -I$(SGX_SDK)/include -I$(SGX_RA_TLS) -I$(INCDIR) -I$(BINDIR) \
+	       -O0 -DWOLFSSL_SGX_ATTESTATION -DWOLFSSL_CERT_EXT -DSGX_GROUP_OUT_OF_DATE
```

Comment:
`-O0 -DWOLFSSL_SGX_ATTESTATION -DWOLFSSL_CERT_EXT -DSGX_GROUP_OUT_OF_DATE` are all really required? Partially?

Reply:
I remove this in the makefile, they are not really required.